src/components/ProductCard/ProductCard.tsx

```diff
@@ -17,17 +19,18 @@ const ProductCard: React.FC<ProductCardAttributes> = ({ product, totalCartCounte
         setCartSate(0);
         setTotalCartCounter(totalCartCounter - 1);
     };
-
+    const { theme } = useContext(ThemeContext);
+    const cartSVG = theme === 'light' ? lightCartSVG : darkCartSVG;
     const addToCart = () => {
         setCartSate(1);
         setTotalCartCounter(totalCartCounter + 1);
     };
     return (
         <div className={styles.container}>
             <img className={styles.image} src={product.images[0]} alt={product.title} />
-            <h2 className={styles.title}>{product.title}</h2>
+            <h1 className={styles.title}>{product.title}</h1>
```

Comment:
don't use more that one h1 on a page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?ref=blog.shimin.io